apps/molgenis-components/src/components/forms/InputRefSelect.vue

```diff
@@ -85,7 +83,10 @@ export default {
       required: false,
     },
     filter: Object,
-    refLabel: String,
+    refLabel: {
+      type: String,
+      required: true,
```

Comment:
should probability be optional and be able to render a value without passing the label

Reply:
Okay, then we make it optional. Original default is that it would fabricate a reflabel based on pkey. We can reinstate that functionality. However, principally I think that is not what user expects so I will first check if we can make sure refLabel should be provided.